includes/Renderer/Stories/Renderer.php

```diff
@@ -624,6 +624,21 @@ protected function render_story_with_poster() {
 		}
 	}
 
+	/**
+	 * Allows customization of html attributes in the web stories widget anchor tag loop
+	 * Converts array into escaped inline html attributes
+	 *
+	 * @return void
+	 */
```

Comment:
```suggestion
	/**
	 * Render additional link attributes.
	 *
	 * Allows customization of html attributes in the web stories widget anchor tag loop
	 * Converts array into escaped inline html attributes.
	 *
	 * @since 1.16.0
	 *
	 * @return void
	 */
```

Reply:
Yes it does, we have it currently live and tracking as expected.

I see, it makes sense. We are keeping a close eye on the releases so I will check when this gets sorted to see if somehow helps our tracking as well.

Thank you for clarifying!